devito/dse/backends/basic.py

```diff
@@ -0,0 +1,45 @@
+from sympy import cos, sin
+
+from devito.dse.backends import AbstractRewriter, dse_pass
+from devito.dse.extended_sympy import bhaskara_cos, bhaskara_sin
+from devito.dse.manipulation import common_subexprs_elimination
+
+from devito.interfaces import ScalarFunction
+
+
+class BasicRewriter(AbstractRewriter):
+
+    def _pipeline(self, state):
+        self._eliminate_intra_stencil_redundancies(state)
```

Comment:
I've moved it to other modes for various reasons. Mainly, once time-invariants are hoisted, it doesn't matter anymore if we approximate them or not from a performance POV